mappings/5695e476953cd3f9a7f77c10f879112b070e8ec65cc171d29cef8ca95350434f4c.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "5695e476953cd3f9a7f77c10f879112b070e8ec65cc171d29cef8ca95350434f4c",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.cardano-stake-pool-cologne.com/",
```

Comment:
Hi @SPOOLCOLOGNE,

the website has the following warning:
<img width="753" alt="2" src="https://github.com/cardano-foundation/cardano-token-registry/assets/17981725/8ec74211-4db0-42c9-9a70-fff42616e461">

1. Remove the warning by creating the issue [here](https://github.com/phishfort/phishfort-lists/issues/new?title=[Legitimate%20Site%20Blocked]%20www.cardano-stake-pool-cologne.com&body=https%3A%2F%2Fwww.cardano-stake-pool-cologne.com%2F).
2. Put the policyID somewhere visible (like twitter bio, reward information on the website, footer, or discord) to verify that the token is associated with the project.

Thanks!